subsys/bluetooth/mesh/lightness_internal.h

```diff
@@ -149,6 +149,19 @@ void lightness_srv_change_lvl(struct bt_mesh_lightness_srv *srv,
 			      struct bt_mesh_lightness_status *status,
 			      bool publish);
 
+
+/** @brief Set the lightness without side effects such as storage or publication
+ *
+ *  @param[in] srv     Lightness server
+ *  @param[in] ctx     Message context or NULL
+ *  @param[in] set     Value to set
+ *  @param[out] status Status response
+ */
+void bt_mesh_lightness_srv_set(struct bt_mesh_lightness_srv *srv,
```

Comment:
Isn't `bt_mesh` prefix for public API?

Reply:
I'll respond to this with a second PR :slightly_smiling_face: https://github.com/nrfconnect/sdk-nrf/pull/5170

The functions will still have global linkage, and may conflict with the application at the linker stage. I've had this in my task list for a long time, so I might as well do it now.